cves/CVE-2014-8541.yml

```diff
@@ -344,7 +384,12 @@ mistakes:
     Use those questions to inspire your answer. Don't feel obligated to answer
     every one. Write a thoughtful entry here that those ing the software
     engineering industry would find interesting.
-  answer:
+  answer: |
+    Quite an old bug on this one! In truth, I think some more stringent
+    controls in terms of unit testing as well as testing a larger variety of
+    inputs would benefit in seeing some of the pitfalls. More importantly,
+    these tests need to be kept up-to-date with current standards involved
+    with the codecs' formats.
```

Comment:
I could use a little more here - on what the developer forgot to do.

Reply:
> ....write a full descri-
You mean in more course-relevant terms, I assume? E.g. what security practice did they fail to adhere to / common pitfall?

Reply:
Resolved in upcoming commit